Module-2 | Day-2/CalculateFrequency.js

```diff
@@ -0,0 +1,21 @@
+﻿function calculateFrequency(string) {
+    var stringArr = string.split("");
+    function charFreq(char){
+        var count = 0;
+        for (var i=0; i<stringArr.length; i++){
```

Comment:
please don't use variable name like i,j,k,x,y,z. Name them properly according to what they do in your code.